Compiler supported targets: xtensa-esp32s3-elf

Traceback (most recent call last):
  File "C:\Espressif\frameworks\esp-idf-v5.1.2\components\partition_table\gen_esp32part.py", line 524, in parse_int
    return int(v, 0)
           ^^^^^^^^^
ValueError: invalid literal for int() with base 0: 'Type'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Espressif\frameworks\esp-idf-v5.1.2\components\partition_table\gen_esp32part.py", line 529, in parse_int
    return keywords[v.lower()]
           ~~~~~~~~^^^^^^^^^^^
KeyError: 'type'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Espressif\frameworks\esp-idf-v5.1.2\components\partition_table\gen_esp32part.py", line 183, in from_csv
    res.append(PartitionDefinition.from_csv(line, line_no + 1))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Espressif\frameworks\esp-idf-v5.1.2\components\partition_table\gen_esp32part.py", line 367, in from_csv
    res.type = res.parse_type(fields[1])
               ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Espressif\frameworks\esp-idf-v5.1.2\components\partition_table\gen_esp32part.py", line 415, in parse_type
    return parse_int(strval, TYPES)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Espressif\frameworks\esp-idf-v5.1.2\components\partition_table\gen_esp32part.py", line 531, in parse_int
    raise InputError("Value '%s' is not valid. Known keywords: %s" % (v, ', '.join(keywords)))
gen_esp32part.InputError: Value 'Type' is not valid. Known keywords: app, data

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Espressif\frameworks\esp-idf-v5.1.2\components\partition_table\parttool.py", line 358, in <module>
    main()
  File "C:\Espressif\frameworks\esp-idf-v5.1.2\components\partition_table\parttool.py", line 327, in main
    target = ParttoolTarget(**target_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Espressif\frameworks\esp-idf-v5.1.2\components\partition_table\parttool.py", line 97, in __init__
    partition_table = gen.PartitionTable.from_csv(f.read())
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Espressif\frameworks\esp-idf-v5.1.2\components\partition_table\gen_esp32part.py", line 185, in from_csv
    raise InputError('Error at line %d: %s\nPlease check extra_partition_subtypes.inc file in build/config directory' % (line_no + 1, err))
gen_esp32part.InputError: Error at line 1: Value 'Type' is not valid. Known keywords: app, data
Please check extra_partition_subtypes.inc file in build/config directory
Traceback (most recent call last):
  File "C:\Espressif\frameworks\esp-idf-v5.1.2\components\partition_table\gen_esp32part.py", line 524, in parse_int
    return int(v, 0)
           ^^^^^^^^^
ValueError: invalid literal for int() with base 0: 'Type'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Espressif\frameworks\esp-idf-v5.1.2\components\partition_table\gen_esp32part.py", line 529, in parse_int
    return keywords[v.lower()]
           ~~~~~~~~^^^^^^^^^^^
KeyError: 'type'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Espressif\frameworks\esp-idf-v5.1.2\components\partition_table\gen_esp32part.py", line 183, in from_csv
    res.append(PartitionDefinition.from_csv(line, line_no + 1))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Espressif\frameworks\esp-idf-v5.1.2\components\partition_table\gen_esp32part.py", line 367, in from_csv
    res.type = res.parse_type(fields[1])
               ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Espressif\frameworks\esp-idf-v5.1.2\components\partition_table\gen_esp32part.py", line 415, in parse_type
    return parse_int(strval, TYPES)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Espressif\frameworks\esp-idf-v5.1.2\components\partition_table\gen_esp32part.py", line 531, in parse_int
    raise InputError("Value '%s' is not valid. Known keywords: %s" % (v, ', '.join(keywords)))
gen_esp32part.InputError: Value 'Type' is not valid. Known keywords: app, data

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Espressif\frameworks\esp-idf-v5.1.2\components\partition_table\parttool.py", line 358, in <module>
    main()
  File "C:\Espressif\frameworks\esp-idf-v5.1.2\components\partition_table\parttool.py", line 327, in main
    target = ParttoolTarget(**target_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Espressif\frameworks\esp-idf-v5.1.2\components\partition_table\parttool.py", line 97, in __init__
    partition_table = gen.PartitionTable.from_csv(f.read())
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Espressif\frameworks\esp-idf-v5.1.2\components\partition_table\gen_esp32part.py", line 185, in from_csv
    raise InputError('Error at line %d: %s\nPlease check extra_partition_subtypes.inc file in build/config directory' % (line_no + 1, err))
gen_esp32part.InputError: Error at line 1: Value 'Type' is not valid. Known keywords: app, data
Please check extra_partition_subtypes.inc file in build/config directory
Traceback (most recent call last):
  File "C:\Espressif\frameworks\esp-idf-v5.1.2\components\partition_table\gen_esp32part.py", line 524, in parse_int
    return int(v, 0)
           ^^^^^^^^^
ValueError: invalid literal for int() with base 0: 'Type'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Espressif\frameworks\esp-idf-v5.1.2\components\partition_table\gen_esp32part.py", line 529, in parse_int
    return keywords[v.lower()]
           ~~~~~~~~^^^^^^^^^^^
KeyError: 'type'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Espressif\frameworks\esp-idf-v5.1.2\components\partition_table\gen_esp32part.py", line 183, in from_csv
    res.append(PartitionDefinition.from_csv(line, line_no + 1))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Espressif\frameworks\esp-idf-v5.1.2\components\partition_table\gen_esp32part.py", line 367, in from_csv
    res.type = res.parse_type(fields[1])
               ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Espressif\frameworks\esp-idf-v5.1.2\components\partition_table\gen_esp32part.py", line 415, in parse_type
    return parse_int(strval, TYPES)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Espressif\frameworks\esp-idf-v5.1.2\components\partition_table\gen_esp32part.py", line 531, in parse_int
    raise InputError("Value '%s' is not valid. Known keywords: %s" % (v, ', '.join(keywords)))
gen_esp32part.InputError: Value 'Type' is not valid. Known keywords: app, data

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Espressif\frameworks\esp-idf-v5.1.2\components\partition_table\parttool.py", line 358, in <module>
    main()
  File "C:\Espressif\frameworks\esp-idf-v5.1.2\components\partition_table\parttool.py", line 327, in main
    target = ParttoolTarget(**target_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Espressif\frameworks\esp-idf-v5.1.2\components\partition_table\parttool.py", line 97, in __init__
    partition_table = gen.PartitionTable.from_csv(f.read())
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Espressif\frameworks\esp-idf-v5.1.2\components\partition_table\gen_esp32part.py", line 185, in from_csv
    raise InputError('Error at line %d: %s\nPlease check extra_partition_subtypes.inc file in build/config directory' % (line_no + 1, err))
gen_esp32part.InputError: Error at line 1: Value 'Type' is not valid. Known keywords: app, data
Please check extra_partition_subtypes.inc file in build/config directory
Traceback (most recent call last):
  File "C:\Espressif\frameworks\esp-idf-v5.1.2\components\partition_table\gen_esp32part.py", line 524, in parse_int
    return int(v, 0)
           ^^^^^^^^^
ValueError: invalid literal for int() with base 0: 'Type'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Espressif\frameworks\esp-idf-v5.1.2\components\partition_table\gen_esp32part.py", line 529, in parse_int
    return keywords[v.lower()]
           ~~~~~~~~^^^^^^^^^^^
KeyError: 'type'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Espressif\frameworks\esp-idf-v5.1.2\components\partition_table\gen_esp32part.py", line 183, in from_csv
    res.append(PartitionDefinition.from_csv(line, line_no + 1))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Espressif\frameworks\esp-idf-v5.1.2\components\partition_table\gen_esp32part.py", line 367, in from_csv
    res.type = res.parse_type(fields[1])
               ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Espressif\frameworks\esp-idf-v5.1.2\components\partition_table\gen_esp32part.py", line 415, in parse_type
    return parse_int(strval, TYPES)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Espressif\frameworks\esp-idf-v5.1.2\components\partition_table\gen_esp32part.py", line 531, in parse_int
    raise InputError("Value '%s' is not valid. Known keywords: %s" % (v, ', '.join(keywords)))
gen_esp32part.InputError: Value 'Type' is not valid. Known keywords: app, data

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Espressif\frameworks\esp-idf-v5.1.2\components\partition_table\parttool.py", line 358, in <module>
    main()
  File "C:\Espressif\frameworks\esp-idf-v5.1.2\components\partition_table\parttool.py", line 327, in main
    target = ParttoolTarget(**target_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Espressif\frameworks\esp-idf-v5.1.2\components\partition_table\parttool.py", line 97, in __init__
    partition_table = gen.PartitionTable.from_csv(f.read())
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Espressif\frameworks\esp-idf-v5.1.2\components\partition_table\gen_esp32part.py", line 185, in from_csv
    raise InputError('Error at line %d: %s\nPlease check extra_partition_subtypes.inc file in build/config directory' % (line_no + 1, err))
gen_esp32part.InputError: Error at line 1: Value 'Type' is not valid. Known keywords: app, data
Please check extra_partition_subtypes.inc file in build/config directory
Traceback (most recent call last):
  File "C:\Espressif\frameworks\esp-idf-v5.1.2\components\partition_table\gen_esp32part.py", line 524, in parse_int
    return int(v, 0)
           ^^^^^^^^^
ValueError: invalid literal for int() with base 0: 'Type'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Espressif\frameworks\esp-idf-v5.1.2\components\partition_table\gen_esp32part.py", line 529, in parse_int
    return keywords[v.lower()]
           ~~~~~~~~^^^^^^^^^^^
KeyError: 'type'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Espressif\frameworks\esp-idf-v5.1.2\components\partition_table\gen_esp32part.py", line 183, in from_csv
    res.append(PartitionDefinition.from_csv(line, line_no + 1))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Espressif\frameworks\esp-idf-v5.1.2\components\partition_table\gen_esp32part.py", line 367, in from_csv
    res.type = res.parse_type(fields[1])
               ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Espressif\frameworks\esp-idf-v5.1.2\components\partition_table\gen_esp32part.py", line 415, in parse_type
    return parse_int(strval, TYPES)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Espressif\frameworks\esp-idf-v5.1.2\components\partition_table\gen_esp32part.py", line 531, in parse_int
    raise InputError("Value '%s' is not valid. Known keywords: %s" % (v, ', '.join(keywords)))
gen_esp32part.InputError: Value 'Type' is not valid. Known keywords: app, data

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Espressif\frameworks\esp-idf-v5.1.2\components\partition_table\parttool.py", line 358, in <module>
    main()
  File "C:\Espressif\frameworks\esp-idf-v5.1.2\components\partition_table\parttool.py", line 327, in main
    target = ParttoolTarget(**target_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Espressif\frameworks\esp-idf-v5.1.2\components\partition_table\parttool.py", line 97, in __init__
    partition_table = gen.PartitionTable.from_csv(f.read())
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Espressif\frameworks\esp-idf-v5.1.2\components\partition_table\gen_esp32part.py", line 185, in from_csv
    raise InputError('Error at line %d: %s\nPlease check extra_partition_subtypes.inc file in build/config directory' % (line_no + 1, err))
gen_esp32part.InputError: Error at line 1: Value 'Type' is not valid. Known keywords: app, data
Please check extra_partition_subtypes.inc file in build/config directory
